internal/pgbackrest/postgres.go

```diff
@@ -39,6 +42,21 @@ func PostgreSQL(
 	outParameters.Mandatory.Add("archive_mode", "on")
 	outParameters.Mandatory.Add("archive_command", archive)
 
+	// archive_timeout is used to determine at what point a WAL file is switched,
+	// if the WAL archive has not reached its full size in # of transactions
+	// (16MB). This has ramifications for log shipping, i.e. it ensures a WAL file
+	// is shipped to an archive every X seconds to help reduce the risk of data
+	// loss in a disaster recovery scenario. For standby servers that are not
+	// connected using streaming replication, this also ensures that new data is
+	// available at least once a minute.
+	//
+	// PostgreSQL documentation considers an archive_timeout of 60 seconds to be
+	// reasonable. There are cases where you may want to set archive_timeout to 0,
+	// for example, when the remote archive (pgBackRest repo) is unavailable; this
+	// is to prevent WAL accumulation on your primary.
+	// - https://www.postgresql.org/docs/current/runtime-config-wal.html#GUC-ARCHIVE-TIMEOUT
+	outParameters.Default.Add("archive_timeout", "60s")
```

Comment:
```suggestion
	outParameters.Default.Add("archive_timeout", "60")
```

I don't believe we add the `s`? The PG docs reference above say to use an integer.